components/ws-daemon/pkg/cpulimit/dispatch.go

```diff
@@ -161,7 +162,7 @@ func (d *DispatchListener) sink(id string, limit Bandwidth, burst bool) {
 	d.workspacesBurstCounterVec.WithLabelValues("none").Inc()
 
 	changed, err := ws.CFS.SetLimit(limit)
-	if err != nil {
+	if err != nil && !strings.HasPrefix(err.Error(), "failed to read") {
```

Comment:
How about defining errors in const instead of error messages?

Reply:
Thought it is not possible to define constants if the error message contains dynamic content (like the path in this case), but it turns out you can: https://smyrman.medium.com/writing-constant-errors-with-go-1-13-10c4191617. Not sure if it is worth it in this case. I can do it if you think it's better.

Reply:
From my point of view, it's fine.

Reply:
You are right!